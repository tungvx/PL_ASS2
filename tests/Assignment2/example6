SPACE= 
DOUBLE=  
%
ThisSPACEisSPACEdifficultDOUBLEtoDOUBLEread!!!
Let's test all strange symbols:;?!,.
