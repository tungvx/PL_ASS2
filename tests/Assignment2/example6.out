This is difficult  to  read!!!
Let's test all strange symbols:;?!,.
